SpatialGDK/Source/Private/EngineClasses/SpatialPackageMapClient.cpp

```diff
@@ -10,7 +10,7 @@
 #include "Interop/Connection/SpatialWorkerConnection.h"
 #include "Interop/SpatialReceiver.h"
 #include "Interop/SpatialSender.h"
-#include "Schema/UnrealObjectRef.h"
+#include "UnrealObjectRef.h"
```

Comment:
I think we should keep UnrealObjectRef.h in the Schema folder to indicate this type is defined in schema.

Reply:
It has moved to the engine, so I didn't add one in there, but I guess I could add a folder indirection there instead?

Reply:
Ah ok I just saw that it had been moved to the Engine. Yeah I think if there was something that made it really obvious this is GDK code like `"improbable/UnrealObjectRef.h"` that would be best.